docs/api.md

```diff
@@ -1,622 +1,583 @@
-silently ignored. For example, `log.info('%s', 'a', 'b')` will only log the
-string `a` and ignore the `'b'` parameter.
+#### `onTerminated` (Function)
 
-If the first argument is an object, all its properties will be included in the
-JSON line. The number of available format string tokens and associated
-parameters will be reduced accodringly.
+Default: `(evt, err) => err ? process.exit(1) : process.exit(0)`
 
-<a id="debug"></a>
-## .debug([obj], msg, [...])
+This function will be invoked during process shutdown when `extreme` is set to `true`.
 
-### Parameters:
-+ `obj` (object): object to be serialized
-+ `msg` (string): the log message to write
-+ `...` (*): format string values when `msg` is a format string
+The signature of the function is `onTerminated(eventName, err)`. If you do not specify a function, Pino will
+invoke `process.exit(0)` when no error has occurred, and `process.exit(1)` otherwise. 
 
-### Discussion:
-Log at `'debug'` level the given `msg`. The `msg` may contain up to 10
-[format string tokens][formatstrs] (given that the method accetps up to 11
-arguments total). The subsequent parameters passed will be used to fill in these
-placeholder tokens with the associated value. Any extra parameters will be
-silently ignored. For example, `log.debug('%s', 'a', 'b')` will only log the
-string `a` and ignore the `'b'` parameter.
+If a function is specified it **must** perform only synchronous operations at this point and
+then exit the process.
 
-If the first argument is an object, all its properties will be included in the
-JSON line. The number of available format string tokens and associated
-parameters will be reduced accodringly.
+* See [Extreme mode ⇗](/docs/extreme.md)
 
-<a id="trace"></a>
-## .trace([obj], msg, [...])
+#### `prettyPrint` (Boolean|Object)
 
-### Parameters:
-+ `obj` (object): object to be serialized
-+ `msg` (string): the log message to write
-+ `...` (*): format string values when `msg` is a format string
+Default: `false`
 
-### Discussion:
-Log at `'trace'` level the given `msg`. The `msg` may contain up to 10
-[format string tokens][formatstrs] (given that the method accetps up to 11
-arguments total). The subsequent parameters passed will be used to fill in these
-placeholder tokens with the associated value. Any extra parameters will be
-silently ignored. For example, `log.trace('%s', 'a', 'b')` will only log the
-string `a` and ignore the `'b'` parameter.
+Enables pretty printing log logs. This is intended for non-production
+configurations. This may be set to a configuration object as outlined in the
+[`pino-pretty` documentation](https://github.com/pinojs/pino-pretty).
+The options object may additionally contain a `prettifier` property to define
+which prettifier module to use. When not present, `prettifier` defaults to
+`'pino-pretty'`. Regardless of the value, the specified prettifier module
+must be installed as a separate dependency. 
```

Comment:
You should add an `npm install pino-pretty` line here.

Reply:
ok